charts/linkerd2-cni/templates/cni-plugin.yaml

```diff
@@ -184,6 +184,9 @@ spec:
       annotations:
         {{.Values.createdByAnnotation}}: {{default (printf "linkerd/helm %s" .Values.cniPluginVersion) .Values.cliVersion}}
     spec:
+      {{- if .Values.tolerations }}
+      {{- include "linkerd.tolerations" . | nindent 6 }}
```

Comment:
Since we are adding a `tolerations` value to the CNI chart, I think this should just be:

```suggested
      {{- include ".Values.tolerations" . | nindent 6 }}
```

I don't think we want to use `linkerd.tolerations` here since the tolerations for the CNI plugin may be different from the rest of the Linkerd installation.

Reply:
My understanding is that this will use the template partial defined here: https://github.com/linkerd/linkerd2/blob/main/charts/partials/templates/_tolerations.tpl, which essentially uses `.Values.tolerations` but does additional formatting with trim etc. Do you still want just `.Values.tolerations`?

Reply:
Correct, `linkerd.toleration` is just making reference to the partial template, so it's fine.

Reply:
... and we use a template because otherwise the `tolerations:` part would not be rendered, only its contents.

Reply:
Okay cool 👍